modules/vulnerabilities/unix/misc/jboss/secgen_metadata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<vulnerability xmlns="http://github.com/cliffe/SecGen/vulnerability"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="http://github.com/cliffe/SecGen/vulnerability">
+
+  <name>JBOSS EAP/AS Remoting Unified Invoker RCE</name>
+  <author>Harry J. Hall</author>
+  <module_license>MIT</module_license>
+  <description>JBOSS EAP versions 6.x contain an RCE vulnerability that can be executed by a unauthenticated user.
+    This invovles sending a serialized Java object to JBOSS EAP over a network.</description>
+
+  <type>Java web exploit</type>
+  <privilege>root_rwx</privilege>
```

Comment:
Does Jboss run as root? If so, it would be good to get an entire copy of this module that does not.